panel/src/fiber/index.js

```diff
@@ -235,6 +235,7 @@ export default class Fiber {
 				cache: "no-store",
 				headers: {
 					...this.options.headers(),
+					"Content-Type": "application/json",
```

Comment:
@lukasbestle Does it matter if it is lowercase or not?

Reply:
I just wonder because it's lowercased in `api::request()`.

Reply:
[MDN says](https://developer.mozilla.org/en-US/docs/Web/API/Headers) that `fetch` handles headers case-insensitively, so I guess it doesn't matter.

Reply:
I just tested it to make sure:

```js
fetch('/', {
  headers: {
    "Content-Type": "application/json",
    "content-type": "text/something"
  }
});
```

In Safari 15.6.1, `fetch()` merges the two unfortunately, so the request header becomes `application/json, text/something`.

That's indeed not great as it means that the headers are treated *partly* case-sensitively.

Reply:
What do you suggest? 

- Always use lowercase?
- Is it always to lowercase the keys before merging them? 

Do you have another idea?

Reply:
> Is it always to lowercase the keys before merging them?

I think that's a great idea!

Reply:
We should do it then in `request.js` as well

Reply:
Done ✅